swarm/storage/localstore/mode_put_test.go

```diff
@@ -335,8 +338,10 @@ func benchmarkPutUpload(b *testing.B, o *Options, count, maxParallelUploads int)
 			sem <- struct{}{}
 
 			go func(i int) {
+				_, err := db.Put(context.Background(), chunk.ModePutUpload, chunks[i])
```

Comment:
This is a very minor thing, but defer should be the first line in this anonymous function.

Reply:
missed that. thanks :pray: